UPGRADING

```diff
@@ -281,6 +281,8 @@ PDO_SQLITE:
   . NumberFormatter::ROUND_TOWARD_ZERO and NumberFormatter::ROUND_AWAY_FROM_ZERO
     have been added as aliases for NumberFormatter::ROUND_DOWN and
     NumberFormatter::ROUND_UP to be consistent with the new PHP_ROUND_* modes.
+  . NumberFormatter::ROUND_HALFODD added to complement existing
+    NumberFormatter::ROUND_HALFEVEN functionality.
```

Comment:
Those two lines should be placed below RFC line which refers to upper entry.

Reply:
Thank you for pointing this out. Fixed now.